tests/integration-tests/tests-base/src/main/java/edu/kit/kastel/mcse/ardoco/core/tests/eval/CodeProject.java

```diff
@@ -1,11 +1,10 @@
 /* Licensed under MIT 2023-2024. */
 package edu.kit.kastel.mcse.ardoco.core.tests.eval;
 
+import static edu.kit.kastel.mcse.ardoco.core.tests.eval.ProjectHelper.loadFileFromResources;
```

Comment:
I am not the biggest fan of importing statically as it sometimes makes it harder to figure out where stuff comes from (looks like a class method but isn't). But it is a personal preference and I can also arrange with this way. However, we can discuss if we want something like that or we should enforce non-static imports only